python/whylogs/api/whylabs/variables.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+from typing import Optional
+
+from .auth_file import get_configparser_object
+
+
+class Variables:
+    @staticmethod
+    def get_variable_from_input(variable_name: str) -> str:
+        variable = input(f"What is your {variable_name}? ")
```

Comment:
This isn't good. For API key you should use `getPass` or it'll get saved in the notebook https://docs.python.org/3/library/getpass.html